TypeScript error

react-konva@19.0.4 fails with a TypeScript error when type checking with `skipLibCheck` set to `true` due to the upgraded dependency on `@types-react-reconciler`:

```
./node_modules/react-konva/ReactKonvaCore.d.ts(105,27): error TS2314: Generic type 'Reconciler<Container, Instance, TextInstance, SuspenseInstance, FormInstance, PublicInstance>' requires 6 type argument(s).
```

--- a/ReactKonvaCore.d.ts
+++ b/ReactKonvaCore.d.ts
@@ -102,4 +102,4 @@ export var Arrow: KonvaNodeComponent<Konva.Arrow, Konva.ArrowConfig>;
 export var Shape: KonvaNodeComponent<Konva.Shape, Konva.ShapeConfig>;
 
 export var useStrictMode: (useStrictMode: boolean) => void;
-export var KonvaRenderer: ReactReconciler.Reconciler<any, any, any, any, any>;
+export var KonvaRenderer: ReactReconciler.Reconciler<any, any, any, any, any, any>;

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -6,7 +6,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "jsx": "react",
-    "skipLibCheck": true
+    "skipLibCheck": false
   },
   "include": ["./src/**/*"],
   "files": [

